### PR TITLE
 added dice_loss  #18231

### LINKED ISSUE #18231: dice_loss

### PR DESCRIPTION
**Pull request by zeus2x7**
added dice loss in paddle.nn.functional.loss referring to issue #18231 
close #18231 
sub-task #15053

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by zeus2x7** [comment]
> **[AryanSharma21](/AryanSharma21) ** left a comment

added a closing tag and test function in ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py

**Comment by zeus2x7** [comment]
@Pchatain  Hi Peter Chatain! can you please review the changes?

**Comment by Pchatain** [comment]
Hi @AryanSharma21 , are you in charge of reviewing this PR?
@zeus2x7 Apologies for the delay. From the check formatting it looks like some linting errors are failing, just small ones like missing a newline at the end of each file. Could you fix these so that the CI tests runs pass?

**Comment by zeus2x7** [comment]
@Pchatain can you please check it now?

**Comment by zeus2x7** [comment]
@Pchatain can you please look into it?
